src/signerkey.js

```diff
@@ -89,4 +89,44 @@ export class SignerKey {
 
     return encodeCheck(strkeyType, raw);
   }
+
+  /**
+   * Creates a signed payload address (P...) from a signer (G...) and a payload.
+   *
+   * @param  {string} address  the signer of the signed payload, as a StrKey (G...)
+   * @param  {Buffer} payload  the raw payload that was signed
+   *
+   * @returns {string}   the StrKey-encoded signed payload address (P...)
+   */
+  static composeSignedPayload(address, payload) {
```

Comment:
Can we name this function `encodeSignedPayload`? It fits existing terms used in the SignerKey type and doesn't clash with any existing function. Using a new term `compose` seems unnecessary.